cvmfs/options.h

```diff
@@ -6,7 +6,7 @@
 #define CVMFS_OPTIONS_H_
 
 #include <stdint.h>
-
+#include <map>
```

Comment:
The map header is included twice, it seems.  Please pass by and remind me to tell you about the header inclusion convention.